pythermalcomfort/models/pet_steady.py

```diff
@@ -7,21 +10,22 @@
 )
 
 
+@np.vectorize
 def pet_steady(
-    tdb,
-    tr,
-    v,
-    rh,
-    met,
-    clo,
-    p_atm=1013.25,
-    position=1,
-    age=23,
-    sex=1,
-    weight=75,
-    height=1.8,
-    wme=0,
-):
+    tdb: Union[npt.ArrayLike, float],
+    tr: Union[npt.ArrayLike, float],
+    v: Union[npt.ArrayLike, float],
+    rh: Union[npt.ArrayLike, float],
+    met: Union[npt.ArrayLike, float],
+    clo: Union[npt.ArrayLike, float],
+    p_atm: Union[npt.ArrayLike, float] = 1013.25,
+    position: Union[npt.ArrayLike, Literal[1, 2, 3]] = 1,
+    age: Union[npt.ArrayLike, int] = 23,
+    sex: Union[npt.ArrayLike, Literal[1, 2]] = 1,
+    weight: Union[npt.ArrayLike, float] = 75,
+    height: Union[npt.ArrayLike, float] = 1.8,
+    wme: Union[npt.ArrayLike, float] = 0,
+) -> Union[npt.ArrayLike, float]:
```

Comment:
I am not super familiar with data types, in the pmv function I am using the following.

```
Union[float, int, np.ndarray, List[float], List[int]]
```

can this function also accept the same inputs as the pmv function? If so shall we use the same type annotation?

Reply:
I had another look at the numpy typing - [`npt.Arraylike`](https://numpy.org/devdocs/reference/typing.html#numpy.typing.ArrayLike) includes all types an array can be created from, hence also `float`, `int` (which is a subset of float) so `List[int]` can be omitted when using `List[float]`.
In general using type annotations is nice, especially for auto completion in an IDE, but likely not very useful when not checked with a type-checker like mypy or pyright.